Doc/whatsnew/3.11.rst

```diff
@@ -1818,31 +1818,66 @@ Porting notes for the C API are
 Build Changes
 =============
 
-* Building Python now requires a C11 compiler. Optional C11 features are not
-  required.
-  (Contributed by Victor Stinner in :issue:`46656`.)
-
-* Building Python now requires support of IEEE 754 floating point numbers.
-  (Contributed by Victor Stinner in :issue:`46917`.)
+* CPython now has :pep:`11` :pep:`Tier 3 support <11#tier-3>` for
+  cross compiling to the `WebAssembly <https://webassembly.org/>`_ platforms
+  `Emscripten <https://emscripten.org/>`_
+  (``wasm32-unknown-emscripten``, i.e. Python in the browser)
+  and `WebAssembly System Interface (WASI) <https://wasi.dev/>`_
+  (``wasm32-unknown-wasi``).
+  The effort is inspired by previous work like `Pyodide <https://pyodide.org/>`_.
+  These platforms provide a limited subset of POSIX APIs; Python standard
+  libraries features and modules related to networking, processes, threading,
+  signals, mmap, and users/groups are not available or don't work.
+  (Emscripten contributed by Christian Heimes and Ethan Smith in :gh:`84461`
+  and WASI contributed by Christian Heimes in :gh:`90473`;
+  platforms promoted in :gh:`95085`)
+
+* Building Python now requires:
+
+  * A `C11 <https://en.cppreference.com/w/c/11>`_ compiler.
+    `Optional C11 features
+    <https://en.wikipedia.org/wiki/C11_(C_standard_revision)#Optional_features>`_
+    are not required.
+    (Contributed by Victor Stinner in :issue:`46656`.)
+
+  * Support for `IEEE 754 <https://en.wikipedia.org/wiki/IEEE_754>`_
+    floating point numbers.
+    (Contributed by Victor Stinner in :issue:`46917`.)
+
+  * Support for `floating point Not-a-Number (NaN)
+    <https://en.wikipedia.org/wiki/NaN#Floating_point>`_,
+    as the :c:macro:`!Py_NO_NAN` macro has been removed.
+    (Contributed by Victor Stinner in :issue:`46656`.)
+
+  * A `C99 <https://en.cppreference.com/w/c/99>`_
+    ``<math.h>`` header file providing the
+    :c:func:`!copysign`, :c:func:`!hypot`, :c:func:`!isfinite`,
+    :c:func:`!isinf`, :c:func:`!isnan`, and :c:func:`!round` functions
+    (contributed by Victor Stinner in :issue:`45440`);
+    and a :c:data:`!NAN` constant or the :c:func:`!__builtin_nan` function
+    (Contributed by Victor Stinner in :issue:`46640`).
```

Comment:
Ah, too late. No big deal.

Reply:
Sorry about that. I was wondering about it, but I wanted to be conservative with my initial changes since I wasn't 100% sure. I can submit a PR for you (with you co-authored on the commit) to apply it going forward, or you can if you prefer.

Reply:
Go ahead. I might get to it later, as my task list shrinks.

Reply:
Wait, your task list actually _shrinks_? It seems as if by invisible dark energy that my task list is perpetually expanding...

In any case, I went ahead and opened this as PR #98781, thanks.